web/src/components/Calculator.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
+    <app-form
+      class="calculator-container"
+      v-if="!formResult"
+      @formSubmit="onSubmit"/>
```

Comment:
As HTML attributes are case insensitive, avoid using camel case names inside them, this is true for props but also for events. See [this](https://vuejs.org/v2/guide/components-props.html#Prop-Casing-camelCase-vs-kebab-case).
```suggestion
      @form-submit="onSubmit"/>
```